### DISCUSSION TITLE
Notifications behind a SSO in Web

### DISCUSSION THREAD

**Discussion by carloscuesta** [community/community#15047] [Other Feature Feedback, Questions, & Ideas]
Hello! 👋🏼 

I use GitHub almost every day for personal and work projects. I'm using the same user account for both purposes. 

When I'm working with my personal laptop, I can see pending 👻 notifications in my dashboard that are not available because I'm not logged through the company SSO. 

As a user that has personal and OSS projects, I would like to have this notifications filtered If I'm not logged inside of the organization so I can fully focus on my personal work 😊 

Not a huge issue but I think it would be a great improvement to the notifications dashboard 🙏🏼

Thanks! ❤️ 

## Demo

<img width="1296" alt="Screenshot 2022-04-16 at 16 19 15" src="https://user-images.githubusercontent.com/7629661/163678581-6f89088e-ad85-475b-9a77-5bb7cd8f57c4.png">

**Comment by rcomer** [comment]
I would very much like to dismiss the blue orb that is currently telling me I have a notification.  I can't do anything about it until I'm back in the office next week and logged in with SSO.

**Comment by peterjuras** [comment]
Hi!

I would also like to not see the notifications badge for work notifications if I'm not logged in.
My two reasons:

* Work life balance: Knowing that something happened, but I can't even see what it is drives me nuts on a Sunday where I just want to relax and do some personal coding
* Security: Although not too urgent - I'm able to view information about the company (something my account is involved in has been updated) although I'm not logged in. This leak should be avoided.

I could understand that for some people this could be useful? (To be nudged to log in with SSO if you're currently not) So at the very least an option to disable this feature would be very helpful.

**Comment by shawnadelic** [comment]
I also would very much like to get rid of the useless irrelevant notification badges.

**Comment by rcomer** [comment]
I’ve been considering setting up a second account to separate my internal and open source work, simply because of this notification issue.  However I found that GitHub specifically recommends against that.

> If you aren't required to use a managed user account, GitHub recommends that you use one personal account for all your work on GitHub.com.

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/managing-multiple-accounts

**Comment by rcomer** [comment]
I have proposed an alternative solution that would work for me at #59091.
